modules/govuk/files/usr/local/bin/govuk_spinup

```diff
@@ -85,6 +85,10 @@ echo "GOVUK_APP_ROOT=${GOVUK_APP_ROOT}"
 echo "GOVUK_APP_RUN=${GOVUK_APP_RUN}"
 echo "GOVUK_APP_LOGROOT=${GOVUK_APP_LOGROOT}"
 
+if [ -n "$GOVUK_APP_ULIMIT_NOFILE" ]; then
```

Comment:
This should be `if [ -n "${GOVUK_APP_ULIMIT_NOFILE:-}" ]; then` because of the `set -u` further up